lib/std/linked_list.zig

```diff
@@ -1,4 +1,4 @@
-const std = @import("std.zig");
+const std = @import("std");
```

Comment:
Inside of the standard library itself don't we need to use `std.zig`?

Reply:
Not anymore. And if you do It that way you cant run just this files tests as It pulls in them all.